frontend/src/lib/api/getData.ts

```diff
@@ -156,6 +157,13 @@ export const getNominatedCandidates = ({
           },
           politicalExperience: attr.politicalExperience
         };
+
+        let photoURL = attr?.photo?.data?.attributes?.url;
```

Comment:
For this `attr.photo.data?.attributes.url` should have enough safeguards.

Reply:
At the same time, we should change the Strapi data def starting from line 172 in `getData.type.ts` thus:

```ts
// inside StrapiCandidateData.attributes
    photo: {
      data?: {
        id: number | string;
        attributes: StrapiImageData;
      };
    };
    politicalExperience: string;
  };
}

export interface StrapiImageData extends StrapiImageFormatData {
  alternativeText?: string;
  caption?: string;
  formats: {
    thumbnail?: StrapiImageFormatData;
    small?: StrapiImageFormatData;
    medium?: StrapiImageFormatData;
    large?: StrapiImageFormatData;
  };
}

export interface StrapiImageFormatData {
  ext: string;
  height?: number;
  name: string;
  size?: number; 
  url: string;
  width?: number;
}

```